src/Symfony/Bridge/PhpUnit/README.md

```diff
@@ -39,6 +39,9 @@ Add this bridge to the `require-dev` section of your composer.json file
 (not in `require`) with e.g.
 `composer require --dev "symfony/phpunit-bridge"`.
 
+Make sure that the `bootstrap.php` file is loaded. You can use the `PhpUnitBridge::getBootstrapFilePath()`
```

Comment:
If I read the full readme with the above sentence, I may understand that I first have to install the bridge with composer require, then make sure the b.php file is included by calling the new function. But b.php is declared in the autoload of the composer.json file so it will be included automatically.
So, more basically, I miss the point. Why should I call this fn even when composer is used?